src/api/object_api.js

```diff
@@ -522,7 +525,10 @@ module.exports = {
             reply: {
                 $ref: '#/definitions/object_info'
             },
-            auth: { system: ['admin', 'user', 'viewer'] }
+            auth: {
+                system: ['admin', 'user'],
+                anonymous: true,
```

Comment:
I don't think we should use anonymous access for RPC calls at all. especially not for operations like `read_object_md` and `put_bucket_policy`.
In my opinion we need to authorize anonymous access in the S3 level, and use an internal token for rpc calls